build.ps1

```diff
@@ -289,15 +311,25 @@ End {
                         # The version of nanoserver in CI doesn't have all the changes needed to verify the image
                         $skipVerification = $true
                     }
+                    $buildArgs =  @{
+                        fromTag = $fromTag
+                        PS_VERSION = $psversion
+                        VCS_REF = $vcf_ref
+                        IMAGE_NAME = $firstActualTag
+                    }
+
+                    if($SasUrl)
+                    {
+                        $sasParts = $SasUrl -split '\?'
```

Comment:
I'm assuming all this works, but want to raise the possibility of using `System.Uri` and `System.UriBuilder` just in case they are an option -- might be less brittle